src/routers/handlers/generic.ts

```diff
@@ -54,6 +55,7 @@ export abstract class GenericHandler<T extends BaseViewData> {
         const { signin_info } = req.session?.data ?? {};
         const isSignedIn = signin_info?.signed_in !== undefined;
         this.viewData.isSignedIn = isSignedIn;
+        this.viewData.template = this.getPageName(req);
```

Comment:
Instead of doing half the checking in the function and half in the nunjucks, could we just make the function "getFeedbackLink" and the property "this.viewData.FeedBackLink" and then return the correct page link from the function? Keep the functionality in a nice well contained function.

Reply:
Yep